samples/samples/async_client_sample.py

```diff
@@ -0,0 +1,648 @@
+from google.cloud.spanner_v1.types import TransactionSelector
+from google.cloud.spanner_v1.types import BatchCreateSessionsRequest
+from google.cloud.spanner_v1.types import DeleteSessionRequest
+import asyncio
+from google.cloud.spanner_v1 import keyset
+from google.cloud.spanner_v1 import transaction
+import time
+import tracemalloc
+import argparse
+
+
+# [ FUNCTION for creating async client and spanner client object]
+def getObjects(instance_id, database_id):
+    # Create async client object
+    async_client = spanner.SpannerAsyncClient()
+
+    # Creates spanner client object
+    spanner_client = Client()
+
+    # Creates the instance object associated with instance id
+    instance = spanner_client.instance(instance_id)
+
+    # Creates the database object associated with instance object
+    database_object = instance.database(database_id)
+
+    # Returns async client object and database object
+    return async_client, database_object
+
+# [START async client partition read]
+async def partition_read(database_object, async_client, task_num):
+
+     print("\nStarting task: ", task_num)
+
+     # Create the session object associated with the database object
+     session_object = session.Session(database_object)
+
+     """Create the session."""
+     session_object.create()
+
+     """Request object for Partition Read."""
+     request = PartitionReadRequest(
+         session = session_object.name,
+         transaction = TransactionSelector(
+             begin = TransactionOptions(
+                 read_only = TransactionOptions.ReadOnly(
+                     strong = True,
+                 ),
+             ),
+         ),
+         table = 'Singers',
+         key_set = keyset.KeySet(all_=True)._to_pb(),
+     )
+
+     """Execute Partition Read."""
+     response = await async_client.partition_read(request = request)
+
+     """Prints the response."""
+     print(response)
+
+     print("\nEnding task: ", task_num)
+# [END async client partition read]
+
+# [START async client partition query]
+async def partition_query(database_object, async_client, task_num):
+
+     print("\nStarting task: ", task_num)
+
+     # Create the session object associated with the database object
+     session_object = session.Session(database_object)
+
+     """Create the session."""
+     session_object.create()
+
+     """Request object for Partition Query."""
+     request = PartitionQueryRequest(
+         session = session_object.name,
+         transaction = TransactionSelector(
+             begin = TransactionOptions(
+                 read_only = TransactionOptions.ReadOnly(
+                     strong = True,
+                 ),
+             ),
+         ),
+         sql =  "UPDATE Singers SET FirstName = 'Trivedi' WHERE SingerId = 1003",
+     )
+
+     """Partition the sql statement query."""
+     response = await async_client.partition_query(request = request)
+
+     """Prints the response."""
```

Comment:
Try avoiding self-explanatory comments like this.

Quoting from: go/pystyle#comments

`Never describe the code. Assume the person reading the code knows Python (though not what you're trying to do) better than you do.`

Reply:
Done.